po/fr.po

```diff
@@ -16892,408 +16843,420 @@ msgstr "Changer le répertoire de travail"
 msgid "Execute specified command for each located file after asking user"
 msgstr ""
+"Exécuter la commande spécifiée pour chaque fichier trouvé après accord de "
+"l’utilisateur"
 
 #: /tmp/fish/implicit/share/completions/find.fish:62
-#, fuzzy
 msgid "Print full file names"
-msgstr "Afficher tous les noms"
+msgstr "Afficher les chemins absolus des fichiers"
 
 #: /tmp/fish/implicit/share/completions/find.fish:63
 msgid ""
 "Execute specified command for each located file, in the files directory, "
 "after asking user"
 msgstr ""
+"Exécuter la commande spécifiée pour chaque fichier trouvé, dans le dossier "
+"contenant le fichier, après accord de l’utilisateur"
 
 #: /tmp/fish/implicit/share/completions/find.fish:64
 msgid "Print null separated full file names"
-msgstr ""
+msgstr "Afficher les chemins absolus des fichiers séparés par \\0"
 
 #: /tmp/fish/implicit/share/completions/find.fish:65
-#, fuzzy
 msgid "Print formated data"
-msgstr "Affiche le texte formaté"
+msgstr "Afficher le résultat au format indiqué"
 
 #: /tmp/fish/implicit/share/completions/find.fish:66
 msgid "Do not recurse unless -depth is specified"
-msgstr ""
+msgstr "Ne pas faire de récursion sauf si -depth est spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:67
-#, fuzzy
 msgid "Exit at once"
-msgstr "Quitter le shell"
+msgstr "Quitter immédiatement"
 
 #: /tmp/fish/implicit/share/completions/find.fish:68
-#, fuzzy
 msgid "List file in ls -dils format"
-msgstr "Lister les bogues au format rss"
+msgstr "Lister les fichiers au format ls -dils"
 
 #: /tmp/fish/implicit/share/completions/find.fish:69
-#, fuzzy
 msgid "Negate result of action"
-msgstr "Inverser l'état de sortie de la tâche"
+msgstr "Inverser le résultat de l’action"
 
 #: /tmp/fish/implicit/share/completions/find.fish:70
 msgid "Result is only true if both previous and next action are true"
 msgstr ""
+"Le résultat n’est vrai que si les actions précédente et suivante sont toutes "
+"deux vraies"
 
 #: /tmp/fish/implicit/share/completions/find.fish:71
 msgid "Result is true if either previous or next action are true"
-msgstr ""
+msgstr "Le résultat est vrai si l’action précédente ou suivante est vraie"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:1
-#, fuzzy
 msgid "Run fish with this command"
-msgstr "Définir ou obtenir la ligne de commande"
+msgstr "Exécuter la commande spécifiée au lieu d’afficher l’invite"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:4
 msgid "Only parse input, do not execute"
 msgstr ""
+"Ne faire que la vérification syntaxique de la commande, pas son exécution"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:5
-#, fuzzy
 msgid "Run in interactive mode"
-msgstr "Exécuter en mode noninteractif"
+msgstr "Exécuter en mode interactif"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:6
-#, fuzzy
 msgid "Run in login mode"
-msgstr "Exécuter en mode fictif"
+msgstr "Exécuter fish comme un shell de connexion"
```

Comment:
Inconsistent terminology for (login) shell:
- line 3707: “login shell” → “invite par défaut” (literally: “default prompt”)
- line 5408: “shell” → “invite de commandes” (literally: “command prompt”)
- line 10548: “login shell” → “invite de commande par défaut” (literally: “default command  prompt”)
- lines 211, 217, current line: “login shell / (shell in) login mode” → “shell de connexion”

Note the latter is in line with other usagesp of shell in the translations.

Reply:
Confusion about shell and command prompt, sorry.